src/task/workload.go

```diff
@@ -136,6 +148,39 @@ func newRandomWorkload(itemNames []string, batchSize int) Workload {
 	}
 }
 
+func newWebWorkload(itemNames []string, numPatterns int, minPatternLength int, maxPatternLength int, replicationFactor int) Workload {
```

Comment:
We want web patterns to sleep for some random period of time in between access patterns - can you do that?